src/mixins/document-mixin.ts

```diff
@@ -70,6 +70,55 @@ export default class DocumentMixin extends Vue {
       })
   }
 
+  /**
+   * Downloads a document from Minio and prompts browser to open/save it.
+   * @param documentKey the document key
+   * @param documentName the document filename
+   * @returns a promise to return the axios response or the error response
+   */
```

Comment:
This is almost the same as "download()" in the other UIs, but those are for Legal API documents. This one is for Minio documents (hence its location in the Document Mixin). Also, this API call passes the document key instead of the URL.